docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -360,7 +360,7 @@ Specifying ``EXPORT_BLOB`` and ``BLOB_PATH`` parameters works similarly to ``CAC
 
             .. code-block:: python
 
-               pipeline_config = { "BLOB_PATH": ".npucache\\compiled_model.blob" }
+               pipeline_config = { "BLOB_PATH": ".npucache\\compiled_model.blob", "WEIGHTS_PATH": "model_path\\openvino_model.bin" }
```

Comment:
This section refers to importing for the blob **with weights**. The export was done with `"CACHE_MODE", "OPTIMIZE_SPEED"` which means the weights are already inside the blob. Above you can find import/export for weightless blob which does require `WEIGHTS_PATH`